src/System.Memory/tests/Performance/Perf.Utf8Parser.cs

```diff
@@ -157,18 +665,19 @@ private static void ParserUInt32(string text)
                 {
                     for (int i = 0; i < Benchmark.InnerIterationCount; i++)
                     {
-                        Utf8Parser.TryParse(utf8ByteSpan, out uint value, out int bytesConsumed);
+                        Utf8Parser.TryParse(utf8ByteSpan, out byte value, out int bytesConsumed);
                         TestHelpers.DoNotIgnore(value, bytesConsumed);
                     }
                 }
             }
         }
 
         [Benchmark(InnerIterationCount = InnerCount)]
-        [InlineData("12837467")] // standard parse
-        [InlineData("-9223372036854775808")] // min value
-        [InlineData("9223372036854775807")] // max value
-        private static void ParserInt64(string text)
+        [InlineData("4212")] // standard parse
+        [InlineData("-32768")] // min value
+        [InlineData("32767")] // max value
+        [InlineData("000000000000000000001235abcdfg")]
+        private static void ByteSpanToInt16(string text)
```

Comment:
The test cases here don't match [StringToInt16_Baseline](https://github.com/dotnet/corefx/pull/32026/files#diff-7f5a11a35cf8403d05b6acd67bc83b28R552).

```C#
        [InlineData("0")]
        [InlineData("10737")] // standard parse (keep the same value, 4212 is fine)
        [InlineData("32767")] // max value
        [InlineData("-32768")] // min value
        [InlineData("-127")] // Remove this one, unnecessary.
        [InlineData("2147")] // Remove this one, unnecessary.
        [InlineData("000000000000000000001235")] // add the abcdfg suffix
```